ATTRIBUTION.md

```diff
@@ -17,6 +17,108 @@ This package contains a variety of materials, including data sets and related ma
+#### Destination
+* Modifications to source files(s): None
+* Location: ***prereise/gather/data/remap_ba_area/cb_2020_us_county_500k.zip***
+
+
+#### Retail Service Territory 
+#### Source
+* Name: Electric Retail Service Territory 
+* Author: Homeland Infrastructure Foundation-Level Data (HIFLD)
+* Description: electric retail service territory
+* Source: https://hifld-geoplatform.opendata.arcgis.com/
+* Exact source location: https://hifld-geoplatform.opendata.arcgis.com/datasets/private-non-retail-shipping-facilities/about
+#### Destination
+* Modifications to source files(s): none
+* Location: ***prereise/gather/data/remap_ba_area/cb_2020_us_county_500k.zip***
+
+
+#### Subregions of NYISO
+#### Source
+* Name: subregions of NYISO
+* Author: ArcGIS
+* Description: subregions of NYISO
+* Source: https://www.arcgis.com/
+* Exact source location: https://www.arcgis.com/home/item.html?id=6fd1de467b134f47a607721f23a69f0c
+#### Destination
+* Modifications to source files(s): None for A-F, they are directly downloaded by the program; G_I are made by grouping and splitting county shapes
+* Location: ***prereise/gather/data/remap_ba_area/nyiso2/***
+
+#### CAISO
+#### Source
+* Name: subregions of CAISO
+* Author: CAISO
+* Description: the subregions of CAISO
+* Source: https://www.caiso.com/
+* Exact source location: https://www.caiso.com/Documents/ISO_ContingencyModelingProposal-SCE_Presentation.pdf
+#### Destination
+* Modifications to source files(s): select county shapes from the image map
+* Location: ***prereise/gather/data/remap_ba_area/CA_balancing_authorities/***
+
+#### PJM
+#### Source
+* Name: subregions of PJM
+* Author: PJM
+* Description: the subregions of PJM
+* Source: PJM
+* Exact source location:  https://www.monitoringanalytics.com/reports/PJM_State_of_the_Market/2018/2018-som-pjm-volume2-appendix.pdf
+#### Destination
+* Modifications to source files(s): The shape files are made from the image.
+* Location: ***prereise/gather/data/remap_ba_area/pjm6.zip***
+
+#### ISONE
+#### Source
+* Name: subregions of ISONE
+* Author: ISONE
+* Description: the subregions of ISONE
+* Source: ISONE
+* Exact source location: https://www.iso-ne.com/isoexpress/web/charts
+#### Destination
+* Modifications to source files(s): the state maps are used
+* Location: ***prereise/gather/data/remap_ba_area/isone.zip***
+
+#### ERCOT
+https://www.ercot.com/files/docs/2021/12/23/2021_Report_Existing_Potential_Electric_System_Constraints_Needs.pdf
+#### Source
+* Name: subregions of ERCOT
+* Author: ERCOT
+* Description: the subregions of ERCOT
+* Source: https://www.ercot.com
+* Exact source location: https://www.ercot.com/news/mediakit/maps/index.html
+#### Destination
+* Modifications to source files(s): the county map is picked to form the subregions based on the ERCOT image.
+* Location: ***prereise/gather/data/remap_ba_area/ercot_zone_mapping.csv***
```

Comment:
Is there a reason to keep this data in its on CSV file, while the data from the MISO and SWPP maps are incorporated into the **BA_County_map.json** file?

Reply:
The csv should be merged to the ba_county_map.josn file to simplify thins.  The change is pushed.